akvo/rsr/tests/iati_checks/test_iati_checks_fields_results.py

```diff
@@ -93,6 +93,7 @@ def test_iati_checks_fields_results_indicator_error(self):
             type=QUANTITATIVE,
             ascending=True,
             description="Description",
+            baseline_value="0",
```

Comment:
Is this the `NO_VALUE` constant? If so, I think it should be used here.

Reply:
this has a different meaning, it just happens that the value is the same.